vera.galstyan/repo/Lesson_3/exercise_04/exercise_04.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python3
+
+#1
+a=(512-282)
+b=(47*48+5)
+
+
+#2
+print(a/b)
```

Comment:
RE: Your output should be 0.1017 not like this 0.1017249004865104. 
Hint: Use print function two decimal precision. You can find a use in google.